metaworld/envs/mujoco/sawyer_xyz/sawyer_bin_picking.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         goal_high = self.hand_high
 
         self.liftThresh = liftThresh
-        self.max_path_length = 150
+        self.max_path_length = 200
```

Comment:
doesn't this and the obj_low/obj_high change mean you need to make this a v2 environment?